src/services/crypto.service.ts

```diff
@@ -8,10 +8,12 @@ import { EncryptedObject } from '../models/domain/encryptedObject';
 import { SymmetricCryptoKey } from '../models/domain/symmetricCryptoKey';
 import { ProfileOrganizationResponse } from '../models/response/profileOrganizationResponse';
 
+import { LogService } from '..//abstractions/log.service';
```

Comment:
why the double slash?

Reply:
This was a typo from deleting `cli` from the path, should be fixed now.